Helper/Price.php

```diff
@@ -124,23 +124,36 @@ public function getProductPrice( // @codingStandardsIgnoreLine
                         );
                     } else {
                         $productType = $product->getTypeInstance();
-                        $childProducts = $productType->getChildrenIds(
-                            $product->getId()
-                        );
-                        $listPrice = 0;
-                        foreach ($childProducts as $skuIds) {
-                            if (is_array($skuIds)) {
-                                try {
-                                    $sku = $this->nostoProductRepository->getByIds($skuIds)->getItems();
-                                    $sku = reset($sku);
-                                    $listPrice += $this->getProductDisplayPrice(
-                                        $sku
-                                    );
-                                } catch (\Exception $e) {
-                                    $this->_logger->error($e->__toString());
+                        $options = $productType->getOptions($product);
+                        $allOptional = true;
+                        $minPrices = [];
+                        $requiredMinPrices = [];
+                        foreach ($options as $option) {
+                            $selectionMinPrice = null;
+                            /* @var Product $selection */
+                            foreach ($option->getSelections() as $selection) {
```

Comment:
I am not sure about magento2. But in magento1, if the option is not available( out of stock), than it is not being considering in the price calculation. https://github.com/Nosto/nosto-magento/pull/377/files#diff-de1e1ce4da97152934687e9a89a70a04R267

Reply:
I tested this and if an selection (sku) is not available or is out of stock it will not be returned when calling `getSelections()`

Reply:
Ok. May be we don't have to check it in magento1 as well.